### PR TITLE
Remnant culture mission on sign fonts

### PR DESCRIPTION
**Pull request by williaji**
## Summary
This is a non-interactive culture mission where the player visits the Remnant outfitter and is given a set of fonts that display the simplified sign language used in spaceport messages and hails.

## Save File
[Fred Flintstone.txt](https://github.com/endless-sky/endless-sky/files/8501882/Fred.Flintstone.txt)
Take off and land on Viminal. Check spaceport.

## Testing Done
Tested as above

### CODE REVIEW

File: data/remnant/remnant 1 introduction.txt [hunk 1]
```diff
@@ -3741,4 +3741,23 @@ mission "Remnant: Expanded Horizons Storms 1"
...
+
+mission "Remnant: Sign Fonts"
+	invisible
+	minor
+	source "Viminal"
+	to offer
+		has "Remnant: Learn Sign Follow Up: offered"
+	on offer
+		conversation
+			`You almost never enter a Remnant outfitter from the public entrance, the way you would in human space. You come through the repair bays dockside, past the technicians working on an ibis with an asterioid scanner and a couple of inhibitor cannons stacked next to it. You are walking along side a hover cart with a triple plasma core and some heat shunts that are destined for the refurbishing team. Once receiving has paid you for the salvage, you head past the small break room with the worn seating but very good coffee machine and into the small showroom.`
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:3755]:
```suggestion
			`You almost never enter a Remnant outfitter from the public entrance, the way you would in human space. You come through the repair bays dockside, past the technicians working on an Ibis with an asteroid scanner and a couple of Inhibitor Cannons stacked next to it. You are walking along side a hover cart with a Korath plasma core and some heat shunts that are destined for the refurbishing team. Once receiving has paid you for the salvage, you head past the small break room with the worn seating but very good coffee machine and into the small showroom.`
```
Paid me for the salvage? Why would I rarely enter through the public entrance for Remnant outfitters alone?

File: data/remnant/remnant 1 introduction.txt [hunk 2]
```diff
@@ -3741,4 +3741,23 @@ mission "Remnant: Expanded Horizons Storms 1"
...
+			`	There are a few open workstations with terminals and holo displays that allow you to review specifications and technical manuals on all the systems for sale, as well as a fit checker that allows you to pull up your ship and verify that it will all fit. Once you are happy with the order, you specify your landing pad and the crew gets to work. You've just pulled up the <ship> on the display when Aeolus walks up.`
+			`	"I have beeen watching you sign with the crews," he says, "and I thought of something that might help you. May I?"`
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:3757]:
```suggestion
			`	"I have been watching you sign with the crews," he says, "and I thought of something that might help you. May I?"`
```

File: data/remnant/remnant 1 introduction.txt [hunk 3]
```diff
@@ -3741,4 +3741,23 @@ mission "Remnant: Expanded Horizons Storms 1"
...
+			`	You move over and let him have the terminal. He moves through several layers of the software you didn't know existed until he finds a long list of communication options. He selects one that looks like two hands and places the order.`
+			`	"This is a display font," he explains. "It will display flight control and hail messages to your ship in the same way as the spaceport displays do. You can also sync it down to your commlink if you wish."`
+			`	He pulls out his own commlink and shows you a notification in the stylised lines marking arm and hand positions that you've come to recognize from exit signs, toilets, departure notices, and civil defense messages.`
```
Comment (Amazinite) [inline data/remnant/remnant 1 introduction.txt:3760]:
```suggestion
			`	He pulls out his own commlink and shows you a notification in the stylized lines marking arm and hand positions that you've come to recognize from exit signs, toilets, departure notices, and civil defense messages.`
```

**Review by Amazinite — COMMENTED** [review]
This conversations seems rather odd to me. The start of the conversations talks about the player entering an outfitter and makes it sound like you just got finished selling salvage and refitting your ship. The problem is that this conversation will offer in the spaceport, not the outfitter, and the player may not have sold any salvage or refitted their ship, so the narrative is saying things happened that the player may not have actually experienced.

Even if the start of the conversation was changed to better fit how this mission actually offers, what happens after that doesn't make much sense to me. It's describe that the player recognizes the symbols from signs they've seen around Remnant spaceports. But this is a Remnant terminal that the player is using, is it not? Why wouldn't this terminal already be in the same configuration as everything else in the spaceport? 

It also seems strange to me that the font is described as showing the hand signs that one would make. Now this is ultimately down to Zitchas if this is the case, but I imagine that Remnant writing wouldn't just be symbols of their hand gestures. It's not as if people who use sign language IRL write in sign language.

So basically every part of this conversation seems off to me; it'd basically need to be completely rewritten to address the above comments.
